### ISSUE TITLE
Tag and text

### ISSUE THREAD

**Issue by Nemecsek** [nowthis/sankeymatic#35]
Now each node has a tag used as link reference: `Node1 [100] Node2`

`Node1` and `Node2` are shown on the plot.

But what is I want to read something else on the graph, instead of the node tag? 

I would like, for example, to be able to set the nodes as:

```
Node1 [100] Node2

:Node1 $text_to_show_for_node1
:Node2 $text_to_show_for_node2
```

It would even be nice if text could be **multiline**, with the first line always bold, and the following lines normal. This would allow to show any kind of parameters/values associated to the node (first line being the id we chose, and the following lines the values). A markup language to define the text to show would make everything very flexible.

Alas I don't "speak" Javascript so it is well above my skills to add this feature.

Or is there a workaround I didn't think about?

**Comment by nplatis** [comment]
I agree that this would be a useful feature. In some cases, nodes may have the same text (most probably, in different columns), which is currently not possible (I think).

**Comment by galbarm** [comment]
upvoting this

**Comment by nowthis** [comment]
@Nemecsek Hello! I have added most of what you described, and you can use it immediately at [sankeymatic.com/build/](https://sankeymatic.com/build/).

You can now customize what text appears on the diagram for any node, regardless of what its 'internal' name is in your input data.

And nodes can now share the **same** display name if you want - the underlying nodes still have to have unique IDs, but you can now show the same label on the diagram for all of them if you want.

> Aside: Sankeymatic **has** had the ability to do multi-line labels for a long while now – just include `\n` in the node name anywhere you want a line break.
> It can feel a bit awkward to have to repeat a formatted label when it is part of several flows, though.
> The new feature provides a better alternative, where you only have to specify the line breaks **once**, when you define the custom label.

The new syntax to do what you're looking for in your original comment looks like this:
```
Node1 [100] Node2

node Node1
.label My multi-line\nlabel for Node 1

node Node2
.label My multi-line\nlabel for Node 2
```

See issue #8 and its comments for further details.

I do not yet have an answer for additional formatting **within** labels (like bold, italic, larger, smaller).
But custom labels and multilines are working.
